scripts/version-updater.gradle

```diff
@@ -47,11 +65,60 @@ class UpdateCoreSdkVersion extends DefaultTask {
 
     @TaskAction
     void write() {
-        Properties gradleProperties = PropertiesIO.readProperties(getProject().gradlePropertiesFilePath)
-        gradleProperties[getProject().gliaCoreSdkVersionTagInProperties] = newCoreSdkVersion
-        PropertiesIO.saveProperties(gradleProperties, getProject().gradlePropertiesFilePath)
+        getProject().saveProperty(getProject().gliaCoreSdkVersionTagInProperties, newCoreSdkVersion)
+    }
+}
+
+/**
+ * A task class for editing the Widget SDK version in version.properties
+ * Example of usage from terminal:
+ * ./gradlew saveWidgetsVersionName --versionCode=77 --versionName=0.77.2
+ */
+class UpdateProjectVersion extends DefaultTask {
+    private String versionName
+    private String versionCode
+
+    @Option(option = "versionName", description = "Widget SDK version name that should be overriden in configuration files")
+    UpdateProjectVersion setVersionName(String versionName) {
+        this.versionName = versionName
+        return this
+    }
+
+    @Option(option = "versionCode", description = "Widget SDK version code that should be overriden in configuration files")
+    UpdateProjectVersion setVersionCode(String versionCode) {
+        this.versionCode = versionCode
+        return this
+    }
+
+    @TaskAction
+    void write() {
+        Map<String, String> propertiesMap = new HashMap<>();
+        propertiesMap.put(getProject().wdigetsVersionNameTagInProperties, versionName)
+        propertiesMap.put(getProject().wdigetsVersionCodeTagInProperties, versionCode)
+        getProject().saveProperties(propertiesMap)
     }
 }
 
 tasks.register('saveCoreSdkVersion', UpdateCoreSdkVersion)
+tasks.register('saveWidgetsVersionName', UpdateProjectVersion)
+
+task bumpVersionMinorLevel {
```

Comment:
Do you think it would be useful to add how this should be used or it's such a well known thing that every Android developer would know how? I am asking because when I was working on adding this to Bitrise, I saw these tasks and didn't know how to use them, so I Googled it. However, I am not an Android developer, so maybe having an example is unnecessary.

Reply:
Fixed: https://github.com/salemove/android-sdk-widgets/compare/1184845c534e842ae4036af38a50e735e8c14be2..adc0b3d4f881681adf33c2e97d9f03f228f53c35